lib/object.gi

```diff
@@ -384,7 +384,13 @@ InstallMethod( PrintObj,
     true,
     [ HasName ],
     SUM_FLAGS, # override anything specific
-    function ( obj )  Print( Name( obj ) ); end );
+    function( obj )
```

Comment:
@ThomasBreuer perhaps less controversial: just remove this method, and add back dedicated printing methods for `Rationals` and other objects relying on this. This assumes there are only a few (Rationals, Integers, GaussianRationals, ... come to mind).

If there are sp many as to make this annoying, we could also introduce a new attribute `NameForPrinting` and use that in here.

---

That said, I wonder how badly this PR breaks package CI tests. Perhaps you can update and rebase this PR (resp. merge current `master` into it), then I could trigger a run of the PackageDistro CI tests against this pull request to see where it stands.

Reply:
O.k., I have updated the pull request accordingly.